analyzers/its/SonarAnalyzer.Testing.ImportBefore.targets

```
@@ -3,7 +3,8 @@
           AfterTargets="ResolveCodeAnalysisRuleSet"
           BeforeTargets="CoreCompile">
     <PropertyGroup>
-      <SonarAnalyzerRegressionTestRunning Condition="Exists('$(MSBuildStartupDirectory)\binaries\SonarAnalyzer.dll')">true</SonarAnalyzerRegressionTestRunning>
+      <BinariesFolder>$(MSBuildStartupDirectory)\..\binaries\</BinariesFolder>
```

Because the next line has it's own slash `$(BinariesFolder)\SonarAnalyzer.dll`
```suggestion
      <BinariesFolder>$(MSBuildStartupDirectory)\..\binaries</BinariesFolder>
```

---

Not really a problem but sure, I can remove it.